assets/src/application/ui/display/ItemCount/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Badge } from 'antd';
+import { BadgeProps } from 'antd/lib/badge';
+import classNames from 'classnames';
+
+import './style.scss';
+
+interface ItemCountProps extends BadgeProps {
+	children?: React.ReactNode;
```

Comment:
You don't need to add `children` here when the type is `React.FC`

Reply:
updated, thx